packages/social/config/default.json

```diff
@@ -5,6 +5,7 @@
     "loginDisabled": false,
     "logo": "./logo.svg",
     "apiServer": "https://${VITE_SERVER_HOST}:${VITE_SERVER_PORT}",
+    "corsServer": "https://${VITE_SERVER_HOST}:${VITE_CORS_SERVER_PORT}",
```

Comment:
I don't think this is necessary. social is our ARC-related codebase, which is getting pulled out very soon and made into a separate XREngine project.